lua/nvim-lsp-installer/servers/r_language_server/init.lua

```diff
@@ -0,0 +1,16 @@
+local server = require 'nvim-lsp-installer.server'
+local shell = require "nvim-lsp-installer.installers.shell"
+local path = require "nvim-lsp-installer.path"
+
+local root_dir = server.get_server_root_path("r_language_server")
+
+return server.Server:new {
+    name = "r_language_server",
+    root_dir = root_dir,
+    installer = shell.polyshell([[R -e 'install.packages("languageserver", repos="http://cran.us.r-project.org")']]),
```

Comment:
This brings some extra custom code but! We can avoid going through the shell as the "mediator" by spawning a process directly instead, like:

```lua
local process = require("nvim-lsp-installer.process")
-- ...
    installer = function (server, callback, context)
        process.spawn("R", {
          args = { "-e", "'install.packages("languageserver", repos="http://cran.us.r-project.org")' },
          stdio_sink = context.stdio_sink
        }, callback)
    end
```

This will go through Lua APIs exclusively (using the `vim.loop (libuv)` API)